openff/benchmark/cli.py

```diff
@@ -777,6 +777,41 @@ def plots(input_path, output_directory):
 
     draw.plot_compare_ffs(input_path, output_directory)
 
+@report.command()
```

Comment:
Can you add docstrings for each of these four commands? See the other commands in the CLI for inspiration on how to structure this.